### ISSUE TITLE
Memory leak?

### ISSUE THREAD

**Issue by wysohn** [ME1312/SubServers-2#63]
### Platform Information
13:50:23 [INFO] CONSOLE executed command: /sub version
13:50:23 [INFO] SubServers > These are the platforms and versions that are running SubServers.Bungee:
13:50:23 [INFO]   Linux 5.4.0-72-generic [x64],
13:50:23 [INFO]   Java 11.0.11 [x64],
13:50:23 [INFO]   BungeeCord git:BungeeCord-Bootstrap:1.16-R0.4-SNAPSHOT:9c078b7:1539,
13:50:23 [INFO]   SubServers.Bungee v2.16.4a (20w46b)
13:50:23 [INFO] 
13:50:23 [INFO] SubServers.Bungee v2.17a is available. You are 1 version behind.

Yes, I know that I'm 1 version behind, but I haven't found this issue being addressed in the latest version as well, so I'm reporting it anyway. Sorry if you already fixed it! The crash happens very rarely and without any context, so it's very hard to catch it.

### What Happened
Server hard crash with OutOfMemoryError

### How It Happened
It usually works fine for about 6~7 days, and it crashes

### Additional Information
```
One instance of net.ME1312.SubData.Server.SubDataClient loaded by jdk.internal.loader.ClassLoaders$AppClassLoader @ 0x80100150 occupies 1,532,034,448 (72.98%) bytes.

Keywords

net.ME1312.SubData.Server.SubDataClient
jdk.internal.loader.ClassLoaders$AppClassLoader @ 0x80100150
```

![Screenshot from 2021-07-03 13-59-54](https://user-images.githubusercontent.com/8789836/124366790-1cbd9380-dc07-11eb-9d87-03d02d9000a3.png)
![Screenshot from 2021-07-03 14-00-22](https://user-images.githubusercontent.com/8789836/124366791-1e875700-dc07-11eb-8756-8043059e1c03.png)
![Screenshot from 2021-07-03 14-00-31](https://user-images.githubusercontent.com/8789836/124366793-20511a80-dc07-11eb-9cf6-cc54cdcd22a2.png)
![Screenshot from 2021-07-03 14-00-48](https://user-images.githubusercontent.com/8789836/124366795-221ade00-dc07-11eb-86e8-ded89ecca3d8.png)

I don't know why would SubDataClient have to use such a high percentage of the heap space, and I don't even utilize the SubData anyway. So I highly suspect that it is a memory leak.

Also, the data is being transferred locally, I don't really think it's something to do with denial of attack.

**Comment by ME1312** [comment]
You're right on the money with this one. I received similar reports and was able to get this fixed by release (2.17). The reason it didn't make the release notes is because the problem only existed in the snapshot versions.

**Comment by wysohn** [comment]
> You're right on the money with this one. I received similar reports and was able to get this fixed by release (2.17). The reason it didn't make the release notes is because the problem only existed in the snapshot versions.

Nice! All I have to do is updating it to the latest version then. Thanks for letting me know!
